packages/synapse-interface/slices/bridgeSlice.ts

```diff
@@ -93,6 +95,12 @@ export const bridgeSlice = createSlice({
     setDestinationAddress: (state, action: PayloadAction<Address | null>) => {
       state.destinationAddress = action.payload
     },
+    addBridgeTxHash: (state, action: PayloadAction<string>) => {
+      state.bridgeTxHashes = [...state.bridgeTxHashes, action.payload];
+    },
+    setBridgeTxHashes: (state, action: PayloadAction<string[]>) => {
```

Comment:
can remove this if we don't use it / export it

Reply:
#2497da4d9330ab7886d18fb6f93e1f211caee9e2